src/Compat.jl

```diff
@@ -931,6 +931,21 @@ if VERSION < v"1.1.0-DEV.472"
     isnothing(::Nothing) = true
 end
 
+# https://github.com/JuliaLang/julia/pull/29749
+if VERSION < v"1.1.0"
```

Comment:
```suggestion
if VERSION < v"1.1.0-DEV.792"
```
(Obtained with `./contrib/commit-name.sh 6b04291` within a jula source tree, where `6b04291` is the respective merge commit.)

Reply:
OK! Should I also change it to `v"7.0" >= VERSION < v"1.1.0-DEV.792"` since `6.0` won't work with it?

Reply:
Good point, although the keyword argument part is a parser thing, so needs to be `@static if` then, unless I'm mistaken.

Reply:
I didn't know about `@static`, thank you. This bypasses the `dims` thing altogether.